modules/web/js/ballerina/components/import-declaration-expanded.jsx

```diff
@@ -17,15 +17,21 @@
  */
 import React from 'react';
 import './import-declaration-expanded.css'
+import ImageUtil from './image-util';
+import PropTypes from 'prop-types';
+import Renderer from './renderer';
 
 export default class importDeclarationExpanded extends React.Component {
     render() {
         const bBox = this.props.bBox;
         const importDeclarationHeight = 30;
-        const importDeclarationWidth = 200;
+        const importInputHeight = 40;
+        const importDeclarationWidth = 300;
         const leftPadding = 10;
+        const topGutter = 10;
+        const iconSize = 20;
         const importElements = [];
-        let lastImportElementY = bBox.y;
+        let lastImportElementY = bBox.y + topGutter;
 
```

Comment:
Shall we unify all these constants (importInputHeight, importDeclarationWidth, etc)  values in to the consts file